lib/pengine/pe_digest.c

```diff
@@ -147,6 +148,9 @@ calculate_main_digest(op_digest_cache_t *data, pe_resource_t *rsc,
     }
 
     action = custom_action(rsc, strdup(key), task, node, TRUE, FALSE, data_set);
+    if (overrides != NULL) {
+        g_hash_table_foreach(overrides, hash2field, data->params_all);
+    }
```

Comment:
Yes, that should work. Just checked how hash2field is handling that - not add and duplicate trace if already there.
Don't know if it is worth the effort but if we add hash2field_override (with matching trace) and do it in the end tracing might be easier to understand / more helpful.
Guess the case where we wouldn't like to override in case it isn't there in the first place isn't interesting ...